file-formats/ntta/README.md

```diff
@@ -3,3 +3,4 @@
 File | Cardinality | Definition | Schema | Example
 :--- | :---  | :--- | :--- | :---
 `<name>.ntta.json` | 1 | [`zif_aff_ntta_v1.intf.abap`](./type/zif_aff_ntta_v1.intf.abap) | [`ntta-v1.json`](./ntta-v1.json) | [`z_aff_example_ntta.ntta.json`](./examples/z_aff_example_ntta.ntta.json)
+`<name>.ntta.json` | 1 | [`zif_aff_ntta_v2.intf.abap`](./type/zif_aff_ntta_v1.intf.abap) | [`ntta-v2.json`](./ntta-v1.json) | [`z_aff_example_ntta.ntta.json`](./examples/z_aff_example_ntta.ntta.json)
```

Comment:
Can you please do 2 tables in the readme as it is done for [ SITO](https://github.com/SAP/abap-file-formats/tree/main/file-formats/sito).